ui-v2/app/templates/dc/acls/-form.hbs

```diff
@@ -26,12 +26,13 @@
     </fieldset>
     <div>
 {{#if create }}
-        <button type="submit" {{ action "create" item}} disabled={{if item.isInvalid 'disabled'}}>Save</button>
+        {{! we only need to check for an empty name here as ember munges autofocus, once we have autofocus back revisit this}}
```

Comment:
Will autofocus fix this? Folks would still be able to hit submit even if the input is auto-focused wouldn't they?

Reply:
When the field is focussed, leaving it fires the validation stuff, hence if I can add autofocus (not just on refresh) it will help - not entirely sure it will completely hedge against every eventuality but it will certainly reduce likelihood. This is just a reminder so that when I add proper autofocus I remember to take some of this out to see whether its still needed.

Reply:
Ahh, I didn't know the bit about validation on blur - would adding validation as part of pre-submission be worth doing too?

Reply:
Vault was using this https://github.com/hashicorp/vault/blob/master/ui/app/mixins/focus-on-insert.js for a bit (not too sure if it's still used) - wouldn't take much to de-jquery-ify it. Also it should probably just be a components `didRender` hook instead of the `Ember.on` stuff that's in there because `Ember.on` is going away I think.

Reply:
Cool, cheers. Might use some of that when I do the autofocus thing. Basically I just want to pick out whatever has `autofocus=""` set and tell ember to focus, basically what the browser does.